### PR TITLE
Create jc.json

### PR DESCRIPTION
**Pull request by sjeeva**
Adding Kothanur CSI church to the map
#8223

### CODE REVIEW

File: _pins/jc.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+githubHandle: jc
+latitude: 13.056898
+longitude: 77.647788
```
Comment (brianamarie) [inline _pins/jc.json]: You will need to add three more `---` to the last line of this file.

File: jc.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: jc
+latitude: 13.056898
+longitude: 77.647788
+---
```
Comment (brianamarie) [inline jc.json]: I think this file will need to be deleted.

**Comment by brianamarie** [comment]
@sjeeva  :wave: Let us know if you want or need any help making these changes. If in a few days the changes still aren't made, one of the GitHub trainers may make the changes for you to keep the repository clean. ✨

File: jc.json (hunk 2 above)
Comment (tx1985) [inline jc.json]: @brianamarie agree

**Pull request merged**